workshops/views.py

```diff
@@ -1514,6 +1516,22 @@ class AwardDelete(RedirectSupportMixin, MockAwardDelete):
     pass
 
 
+class AwardCertification(OnlyForAdminsMixin, AMYDetailView):
+    queryset = Award.objects.all()
+
+    def get_template_names(self):
+        return 'workshops/certificates/{}.svg'.format(self.object.badge.name)
+
+    def render_to_response(self, context, **response_kwargs):
+        return HttpResponse(
+            cairosvg.svg2pdf(
```

Comment:
Hm - are we sure PDF generation will be fast enough that the HTTP response doesn't time out?